docs/src/tutorial/strings.rst

```diff
@@ -651,25 +651,25 @@ Windows and wide character APIs
 -------------------------------
 
 Windows system APIs natively support Unicode in the form of
-zero-terminated UTF-16 encoded :c:type:`wchar_t*` strings, so called
+zero-terminated UTF-16 encoded :c:expr:`wchar_t*` strings, so called
 "wide strings".
 
 By default, Windows builds of CPython define :c:type:`Py_UNICODE` as
-a synonym for :c:type:`wchar_t`. This makes internal :obj:`unicode`
+a synonym for :c:expr:`wchar_t`. This makes internal :obj:`unicode`
```

Comment:
Can't we stick with `:c:type:` here?

Reply:
There is no definition of `wchar_t` type. This is yielding errors during generating of documentation:
```
/Users/matus/dev/cython/docs/src/tutorial/strings.rst:653: WARNING: Unparseable C cross-reference: 'wchar_t*'
Invalid C declaration: Expected end of definition. [error at 7]
  wchar_t*
  -------^
```